wurst/systems/trade/TradeShipCoordinator.wurst

```diff
@@ -0,0 +1,39 @@
+package TradeShipCoordinator
+import TradeShipList
+import TradeShipSupervisor
+import TradeShipRoutePair
+import LinkedList
+
+public class TradeShipCoordinator
+    var nextShip = TradeShipList.getFirst()
+    let routes = new LinkedList<TradeShipRoutePair>()
```

Comment:
How does `var` versus `let` work for objects? You can modify the object but not change the reference?

Reply:
@jonathanmclaus yeah exactly